hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/HapiTokenCreate.java

```diff
@@ -366,6 +357,29 @@ protected Consumer<TransactionBody.Builder> opBodyDef(final HapiSpec spec) throw
                                     b.addCustomFees(supplier.apply(spec));
                                 }
                             }
+                            // We often want to use an existing contract to control the keys of various types (supply,
```

Comment:
I see only one place where the setter `contractKey()` is called, over in `PrecompileMintThrottlingCheck`, correct?  We're going to use it more?  (I mean, seems like a nice facility, but underused right now?)